### ISSUE TITLE
Why this repository isn't provided via Package Manager ?

### ISSUE THREAD

**Issue by alamboley** [Unity-Technologies/2d-extras#39]
Hi,

Everything is in the question! Would be cool to grab all plugins upgrade at the same place ;)

**Comment by janissimsons** [comment]
I agree, it should be downloadable via package manager.

**Comment by badjano** [comment]
Everything on Unity-tech github should be on package manager, I have no idea why they're not

**Comment by RyotaMurohoshi** [comment]
@ChuanXin-Unity 

https://github.com/Unity-Technologies/2d-extras/issues/152#issuecomment-552774057

Hi! According this comment, Unity-Technologies  have plan to add 2x-extras in Package Manager.

Would you tell me plan for adding 2x-extras to Package Manager?

Some user for 2x-extras does not have git environment. It is so good to share 2x-extras for the users if 2x-extras is provided via Package Manager.

**Comment by ChuanXin-Unity** [comment]
Yes, the plan is to have this package be available in the Unity Package Manager. Some changes have been made recently for this to happen.

There are still some internal checks and processes that need to go through before it is made available in the Unity Package Manager. I will update here when they are done!

**Comment by RyotaMurohoshi** [comment]
So great!
I'm looking forward to using 2d-extras via Unity Package Manager!

Thanks a lot!

**Comment by RyotaMurohoshi** [comment]
@ChuanXin-Unity 

Hi! I am looking forward to use 2d-extras via Unity Package Manager.

By the way, are there still some internal checks and processes?
As contributor, are there any things to help publishing 2d-extras to Unity Package Manager?

**Comment by ChuanXin-Unity** [comment]
Thanks for asking! Unfortunately, I believe for now we will need to wait for the internal processes to be done.

This involves factors such as legal, product management, release management, QA etc, which involve quite a few people. This is still being worked on, despite the lack of updates here!

**Comment by RyotaMurohoshi** [comment]
Thank you! I got it.

I am looking forward to use 2d-extras via Unity Package Manager.

**Comment by RyotaMurohoshi** [comment]
@ChuanXin-Unity Hi!

With Unity 2019.4.8f1

I added ednext code in manifest.json

```
"com.unity.2d.tilemap.extras": "1.5.0-preview",
```

With above settings, I can use 2d-extras from Unity registry.

Is it ready 2d-extras for usage via Package Manager?

**Comment by ChuanXin-Unity** [comment]
Hi, this is not publicly as we need to go through some more checks before the package is listed in the Package Manager.

Also, note that the package on Package Manager will not be a 1-1 reproduction of the repository contents, but will only contain a subset of the features. Do let us know if features should be added to the listed package!

**Comment by CoryJones** [comment]
The readme says this is available as a package but I am unable to find it. Is it still in progress?

**Comment by ChuanXin-Unity** [comment]
Hi, this is available in the Unity Package Manager from Unity 2020.1 onwards. As it is a preview package, you may need to enable previews in the Unity Package Manager to see the package. Hope this helps!
